pkgs/applications/networking/p2p/qbittorrent/default.nix

```diff
@@ -33,12 +33,6 @@ stdenv.mkDerivation rec {
     (if webuiSupport then "" else "--disable-webui")
   ] ++ optional debugSupport "--enable-debug";
 
-  # The lrelease binary is named lrelease instead of lrelease-qt4
-  patches = [ ./fix-lrelease.patch ];
-
-  # https://github.com/qbittorrent/qBittorrent/issues/1992
-  enableParallelBuilding = false;
```

Comment:
Parallel building is disabled by default, you have to set it to `true` to enable.